docs/RC/GetInteriorVehicleData/index.md

```diff
@@ -10,29 +10,33 @@ Purpose
+SDL forwards a GetInteriorVehicleData request to HMI only if there is no cached data available for the requested module (`moduleId` + `moduleType`) or it needs to subscribe to/unsubscribe from the module from HMI.  
 Otherwise, SDL responds to the request with the cached data without forwarding it to HMI.
 
-The HMI should only return interior vehicle data that corresponds to the request`moduleType`. For example, if `moduleType = CLIMATE`, only return [`ClimateControlData`](../../common/structs/#climatecontroldata) and do not return [`RadioControlData`](../../common/structs/#radiocontroldata). 
+The HMI should return interior vehicle data that corresponds to the requested module (`moduleId` + `moduleType`).
 
 ### Request  
 
-GetInteriorVehicleData is a request originated by a Remote Control Mobile Application. The HMI should only return interior vehicle data that corresponds to the request module type.  
-For example, if `moduleType = CLIMATE`, only return [`ClimateControlData`](../../common/structs/#climatecontroldata) and do not return [`RadioControlData`](../../common/structs/#radiocontroldata).  
+GetInteriorVehicleData is a request originated by a Remote Control Mobile Application. 
+If the optional `moduleId` is not provided in a GetInteriorVehicleData_request from a Remote Control Mobile Application, and if there is at least one module (published by capabilities) of the same moduleType, SDL core will use the default `moduleId` (the moduleId of first module provided in RC capabilities for requested `moduleType`) when processing the request.
 
 If the parameter `subscribe` is set to true, the mobile application has requested to subscribe to the module data defined by the [`moduleType`](../../common/enums/#moduletype) parameter.
 
 #### Parameters
 
-|Name|Type|Mandatory|Additional|
-|:---|:---|:--------|:---------|
-|moduleType|[Common.ModuleType](../../common/enums/#moduletype)|true||
-|subscribe|Boolean|false|defvalue="false"|
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:---------------|
+|moduleType|[Common.ModuleType](../../common/enums/#moduletype)|true||The type of a RC module to retrieve module data from the vehicle. <br> In the future, this should be the Identification of a module.|
+|moduleId|String|false|maxlength="100"|Id of a module, published by System Capability.|
+|subscribe|Boolean|false|defvalue="false"|If subscribe is true, the head unit will register OnInteriorVehicleData notifications for the requested module (moduleId and moduleType). <br> If subscribe is false, the head unit will unregister OnInteriorVehicleData notifications for the requested module (moduleId and moduleType). <br> If subscribe is not included, the subscription status of the app for the requested module (moduleId and moduleType) will remain unchanged.|
```

Comment:
Probably don't require a `defvalue` attribute here since its not included in the HMI_API https://github.com/smartdevicelink/sdl_core/pull/2984/files#diff-f81183d34277b39fa042fd7e4eed256fR6837
```suggestion
|subscribe|Boolean|false||If subscribe is true, the head unit will register OnInteriorVehicleData notifications for the requested module (moduleId and moduleType). <br> If subscribe is false, the head unit will unregister OnInteriorVehicleData notifications for the requested module (moduleId and moduleType). <br> If subscribe is not included, the subscription status of the app for the requested module (moduleId and moduleType) will remain unchanged.|
```